### PR TITLE
[k8] Fix failing test

### PR DESCRIPTION
**Pull request by dmolokanov**
Fix failing unit tests after merging conflicting changes in 2 concurrent PR.

### CODE REVIEW

File: edgelet/edgelet-kube/src/module/trust_bundle.rs [hunk 1]
```diff
@@ -212,8 +212,8 @@ mod tests {
                     "items": [
                         {
                             "metadata": {
-                                "name": constants::PROXY_CONFIG_TRUST_BUNDLE_NAME,
-                                "namespace": "my-namespace",
+                                "name": "device1-iotedged-proxy-trust-bundle",
```
Comment (philipktlin) [inline edgelet/edgelet-kube/src/module/trust_bundle.rs]: should these be constants and share across tests?

**Comment by philipktlin** [comment]
btw, how this can get into master if tests are failed?

File: edgelet/edgelet-kube/src/module/trust_bundle.rs (hunk 1 above)
Comment (dmolokanov) [reply edgelet/edgelet-kube/src/module/trust_bundle.rs]: Good idea.

**Comment by dmolokanov** [comment]
> btw, how this can get into master if tests are failed?
It was 2 concurrent PR merged at the same time. Should wait until tests passed after first merge next time.

**Pull request merged**